sdk/selfhelp/Azure.ResourceManager.SelfHelp/src/Generated/Models/Filter.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.SelfHelp.Models
+{
+    /// <summary> Filter criterion. </summary>
+    public partial class Filter
```

Comment:
The name is too generic. Same as `Name`, `ResultType`, `Section`, `Type`, `Video`... etc

Reply:
IMO since all these properties are grouped under Filter they are not generic. If they are not grouped under filter we can consider renaming it

Reply:
> IMO since all these properties are grouped under Filter they are not generic. If they are not grouped under filter we can consider renaming it

@sahilpwr What do you mean by `all these properties`? Those names I mentioned are names of types, like ``Name.cs`, `ResultType.cs`...etc

Reply:
@archerzz I am getting little confused. This code is generated from ARM swagger right? ARM already approved our swagger and they were okay with the naming convention. If you want we can sync offline and I would be happy to go over our naming convention.

Reply:
@archerzz updated the names